packages/rocketchat-videobridge/client/tabBar.js

```diff
@@ -0,0 +1,13 @@
+Meteor.startup(function() {
+	Tracker.autorun(function() {
+			RocketChat.TabBar.addButton({
+				groups: ['directmessage','privategroup'],
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A space is required after ','.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2700984384/issues/source?bid=3199634&fileBranchId=3342079#l4)